src/pages/iou/IOUDetailsModal.js

```diff
@@ -211,7 +213,7 @@ class IOUDetailsModal extends Component {
                 {reportIsLoading ? <ActivityIndicator color={themeColors.text} /> : (
                     <View style={[styles.flex1, styles.justifyContentBetween]}>
                         <ScrollView contentContainerStyle={styles.iouDetailsContainer}>
-                            {(this.props.iouReport.hasOutstandingIOU) && (
+                            {(this.props.iouReport.hasOutstandingIOU || isIOUPaid) && (
```

Comment:
Isn't this handled [here](https://github.com/Expensify/Expensify.cash/pull/3404/files#diff-3597ac886dade8ed106d403fad7c534b6a0fd9423bf6081877749b0295fb3bc8R90)?

Reply:
Yeah, good spot. I've removed that condition. Perhaps all the `shouldDisplay` logic should be handled by the component itself, but I don't think that change is necessary as part of this PR given that it's blocking N5. 

Let me know if you disagree.